docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -390,25 +390,27 @@ alter-action is:
 
 column-definition is:
    column-name data-type
-      ([DEFAULT default]
-         [[constraint constraint-name] column-constraint])
+      [[DEFAULT default]
+         [[constraint constraint-name] column-constraint]]
 
 data-type is:
      char[acter] [(length)[characters]]

```

Comment:
This should say, "char[acter] (length [characters])"

Reply:
Thanks for your review @DaveBirdsall 

`length [characters]` is optional.
If `length` is not specified, then default is 1 character.

![default](https://user-images.githubusercontent.com/20532956/42794351-da9d61f2-89b0-11e8-9fca-10f2e09e6511.png)